EECS-581/attendance-tracker
# Create Account Page select user type working

create account page now asks what user you are and is integrated with google sign in. 

To Test:
Run locally 
manually change url to create_account 
signin with google with an account that hasn't been signed in if you have one 

should say new account created user type and then if you refresh page and try again it should just say user signed in
